src/nuxt.ts

```diff
@@ -38,22 +37,18 @@ const resolveRootDir = () => {
 export async function loadFixture () {
   const { options } = getContext()
 
-  options.rootDir = resolveRootDir()
+  const rootDir = options.rootDir || resolveRootDir()
 
   const { loadNuxtConfig } = await loadNuxtPackage()
   options.config = await loadNuxtConfig({
-    rootDir: options.rootDir,
+    rootDir,
     configFile: options.configFile,
     configOverrides: options.config
   })
 
-  if (!options.config.rootDir) {
-    options.config.rootDir = options.rootDir
```

Comment:
`config.rootDir` is passed to nuxt (core) config. `options` is for test-utils context.